djangocms_snippet/cms_plugins.py

```diff
@@ -39,8 +41,11 @@ def render(self, context, instance, placeholder):
             content = _('Template %(template)s does not exist.') % {
                 'template': instance.snippet.template}
         except Exception:
-            exc = sys.exc_info()[0]
-            content = str(exc)
+            typ, val, tb = sys.exc_info()
+            content = '\n<pre>\n' + escape(str(val)) + '\n</pre>\n'
+            if settings.DEBUG:
+                esctbf = map(lambda x: escape(str(x)),traceback.format_tb(tb))
```

Comment:
Would this potentially be a XSS issue if an attacker was able to get some HTML/JS into the traceback since ``traceback.format_tb(tb)`` is not escaped?

Reply:
Oh yes, you are right, mark_safe is very risky. Only html formatting is needed here.
Copy-paste this code? https://github.com/django/django/blob/1.11.5/django/utils/html.py#L48

Reply:
Traceback is removed now! This was useful in order to understand code error in file cms_plugins.py, which was triggerred when introducing menus in snippets, Cf. #35.